packages/api/src/cli/codegen/languages/typescript.ts

```diff
@@ -189,80 +163,114 @@ export default class TSGenerator extends CodeGeneratorLanguage {
-    this.sdk = sdkSource.addClass({
+    this.sdk = sourceFile.addClass({
       name: 'SDK',
-    });
-
-    this.sdkExport = sdkSource.addVariableStatement({
-      declarationKind: VariableDeclarationKind.Const,
-      declarations: [
-        {
-          name: 'createSDK',
-          initializer: writer => {
-            // `ts-morph` doesn't have any way to cleanly create an IFEE.
-            writer.writeLine('(() => { return new SDK(); })()');
-            return writer;
-          },
-        },
+      properties: [
+        { name: 'spec', type: 'Oas' },
+        { name: 'core', type: 'APICore' },
       ],
     });
 
-    /**
-     * There's an annoying quirk with `ts-morph` where if we set the `createSDK` function to be the
-     * default export with `isDefaultExport` then when we compile it to an ES5 target for CJS
-     * environments it'll be exported as `export.default = createSDK`, which when you try to load it
-     * you'll need to run `require('@api/sdk').default`.
-     *
-     * Instead here by plainly creating `createSDK` in the source file and then setting this export
-     * assignment it'll export the SDK IFEE initializer as `module.exports = createSDK` so people
-     * can cleanly load their SDK with `require('@api/sdk)`.
-     *
-     * A whole lot of debugging went into here to let people not have to worry about `.default`
-     * messes. I hope it's worth it!
-     */
-    if (this.compilerTarget === 'cjs') {
-      sdkSource.addExportAssignment({
-        expression: 'createSDK',
-      });
-    } else {
-      /**
-       * Because `createSDK` above is an IFEE constant we can't use `setIsDefaultExport` on it due
-       * to `ts-morph` not having great handling for IFEE's.
-       *
-       * If we were to call `setIsDefaultExport` on our IFEE to attempt to compile it as
-       * `export default createSDK` then `ts-morph` hard crashes with a "Error replacing tree: The
-       * children of the old and new trees were expected to have the same count" exception due to
-       * it not being able properly handle IFEE's. It's for that reason that we need to manually
-       * write a statement expression to set `createSDK` as the default export.
-       *
-       * Another quirk that this work avoids is there being an empty `export {};` at the very end
-       * of our compiled `d.ts` declaration file. I'm not sure why it was being added, and it
-       * didn't appear to be harming anything, but us manually creating this export statement
-       * causes it to go away.
-       *
-       * Thankfully, fortunately, and curiously, these are all only problems in non-CJS compiled
-       * targets. ¯\_(ツ)_/¯
-       */
-      sdkSource.addStatements('export default createSDK');
-    }
-
-    this.sdk.addProperties([
-      { name: 'spec', type: 'Oas' },
-      { name: 'core', type: 'APICore' },
-      { name: 'authKeys', type: '(number | string)[][]', initializer: '[]' },
```

Comment:
Removed this because it wasn't being used anymore. I refactored all of the auth key stuff upstream into `api/core` but forgot to remove this then.